product-backend/scripts/purge-database.sql

```diff
@@ -0,0 +1,15 @@
+DELIMITER $$
+DROP PROCEDURE IF EXISTS drop_procedure $$
+CREATE PROCEDURE drop_procedure()
+BEGIN
+    START TRANSACTION;
+    
+    DROP TABLE IF EXISTS JobRole;
+    
+    
+        COMMIT;
+        SELECT 'TRANSACTION COMMITED SUCCESFULLY';
```

Comment:
This changes are redundant 
```suggestion
BEGIN
    START TRANSACTION;
    
    DROP TABLE IF EXISTS JobRole;
    
    
        COMMIT;
        SELECT 'TRANSACTION COMMITED SUCCESFULLY';
```